client/composables/axios/useAxiosFactory.ts

```diff
@@ -24,10 +24,10 @@ function tryRefreshFactory(axios: AxiosInstance) {
   }
 }
 
-const appConfig = useRuntimeConfig()
-const axiosMiddlewares = useAxiosMiddlewares()
-
 export function useAxiosFactory() {
+  const appConfig = useRuntimeConfig()
+  const axiosMiddlewares = useAxiosMiddlewares()
+
```

Comment:
What was the issue?

Reply:
scoping ; when trying to use a composable which depends on the file which defines useRuntimeConfig, the initializations of the latter comes before the nuxt app is initialized, causing and error. We didn't see it before because we've never been in such a case. It provided only a minor optimization anyway.